definitions/tools/samtools_view_convert2bam.cwl

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env cwl-runner
```

Comment:
Any reason not to use the existing `cram_to_bam.cwl` instead of this? 
https://github.com/genome/analysis-workflows/blob/master/definitions/tools/cram_to_bam.cwl

Reply:
no reason other than I overlooked it when I went looking for it. I can create a one-off script to fill in the input yaml.... it would be a bit hacky but if everyones okay with that, would probably write a bash script to use the gms listers to retrieve the model data dirs and mv files from the patch over.